app/Http/Controllers/ApiController.php

```diff
@@ -18,13 +18,13 @@ public function __construct()
     public function getStudentDetails(Request $request)
     {
         if (!($request->has("username"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing username"));
+            $userResponse = array("status" => "Fail", "message" => "missing username");
```

Comment:
We can actually just call this `$response`. `$userResponse` is misleading. My first impression was that it contained the user's details.